src/foundry/foundry.js/hooks.d.ts

```diff
@@ -628,10 +634,11 @@ declare global {
     type GetPlaylistDirectorySoundContext = (jq: JQuery, entryOptions: ContextMenuEntry[]) => boolean | void;
 
     /**
-     * @remarks This is called after getting the {@link ContextMenu} options for a {@link SidebarDirectory} entry, but
-     * before creating the ContextMenu.
-     * @param jq           - the JQuery of the ContextMenu parent element
-     * @param entryOptions - the already created ContextMenuOptions
+     * A hook event that fires when the context menu for entries in a SidebarTab
+     * is constructed. Substitute the SidebarTab name in the hook event to target
+     * a specific SidebarTab, for example "getActorDirectoryEntryContext".
+     * @param html         - The HTML element to which the context options are attached
```

Comment:
The parameters in the function definition should be renamed according to the documentation.